gef-ui/package.json

```diff
@@ -77,6 +77,7 @@
     "@babel/preset-env": "7.20.2",
     "@babel/runtime": "7.20.7",
     "@types/jest": "^29.2.5",
+    "ajv": "^8.12.0",
```

Comment:
is this package used anywhere?

Reply:
Yes it is being used during `npm run build`.